sisl/io/tbtrans/tbt.py

```diff
@@ -2537,7 +2537,8 @@ def __call__(self, parser, ns, value, option_string=None):
                 else:
                     ns._tbt.write_tbtav(value)
         p.add_argument('--tbt-av', action=AVOut, nargs='?', default=None,
-                       help='Create "{0}" with the k-averaged quantities of this file.'.format(self.file.replace('TBT.nc', 'TBT.AV.nc')))
+                       help='Create "{0}" with the k-averaged quantities of this file.'.format(
+                        str(self.file).replace('TBT.nc', 'TBT.AV.nc')))
```

Comment:
Unfortunately it's not so easy with multiple suffixes, there isn't a `.with_suffixes` method :( It could be `self.file.with_name(self.file.name.replace('TBT.nc', 'TBT.AV.nc'))` but that's hardly more elegant...